src/nvim/terminal.c

```diff
@@ -217,10 +214,14 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
+  // Only relevant in terminal true color mode
+  if (true_color) {
+    vterm_state_set_bold_highbright(state, true);
```

Comment:
@DarkDefender Thanks for covering all of those scenarios. It is clear now we don't need the extra setting.